var/spack/repos/builtin/packages/emacs/package.py

```diff
@@ -49,10 +50,11 @@ class Emacs(AutotoolsPackage, GNUMirrorPackage):
     depends_on('gtkplus', when='+X toolkit=gtk')
     depends_on('gnutls', when='+tls')
     depends_on('jpeg')
-    depends_on('m4', when="@master:")
-    depends_on('autoconf', when="@master:")
-    depends_on('automake', when="@master:")
-    depends_on('libtool', when="@master:")
+    depends_on('m4', type=('build'), when="@master:")
+    depends_on('autoconf', type=('build'), when="@master:")
+    depends_on('automake', type=('build'), when="@master:")
+    depends_on('libtool', type=('build'), when="@master:")
+    depends_on('gcc@11.1.0 +strip languages=jit', when="+native")
```

Comment:
This probably only needs GCC 11 or newer, right? Then `gcc@11:` would be the way to go.